src/cli.ts

```diff
@@ -1,11 +1,14 @@
 import * as childProcess from 'child_process';
 import Process = NodeJS.Process;
+import { Logger } from './logger';
 
 /**
  * The command line interface.
  * Allows to execute command lines on the system.
  */
 export class Cli {
+    private static logger: Logger = Logger.getLogger('dry-dry.Cli');
```

Comment:
Please use dry instead of dry-dry.
The repo and the package are named dry-dry because dry was already in the npm registry.
The cli is dry, so dry prefix should be enough.

Reply:
Ok, done